docs/rules/attributes-order.md

```diff
@@ -17,10 +17,10 @@ ex: 'v-once', 'v-pre'
 ex: 'id'
 - UNIQUE
 ex: 'ref', 'key', 'slot'
-- BINDING
-ex: 'v-model', 'v-bind', ':property="foo"'
+- TWO\_WAY\_BINDING
+ex: 'v-model'
 - OTHER_ATTR
-ex: 'customProp="foo"'
+ex: 'custom-prop="foo"', 'v-bind:prop="foo"', ':prop="foo"', 'v-custom-directive'
```

Comment:
I don't think we should mix other attributes with bindings.
Let's keep both groups: `TWO_WAY_BINDINGS` and `BINDINGS`

Reply:
It's a common situation that when user needs to change a static property value to a dynamic binding value. If we separate them into two groups, the user may have to modify the order of these properties and feel confused. Maybe it's still better to let user themselves to sort these attributes.